dev/AppNotifications/AppNotificationManager.cpp

```diff
@@ -355,4 +358,12 @@ namespace winrt::Microsoft::Windows::AppNotifications::implementation
 
         co_return toastNotifications;
     }
+
+    winrt::Windows::Foundation::IInspectable AppNotificationManager::Deserialize(winrt::Windows::Foundation::Uri const& /* uri */)
```

Comment:
Why take a uri that isn't being used?

Reply:
ExtensionMap forces a specific type winrt::Windows::Foundation::IInspectable(*factory)(winrt::Windows::Foundation::Uri const& uri) as Deserialize. We must at least have the param